dist/Locale-Maketext/lib/Locale/Maketext.pm

```diff
@@ -136,19 +136,21 @@ sub fail_with { # an actual attribute method!
 
 #--------------------------------------------------------------------------
 
-sub blacklist {
-    my ( $handle, @methods ) = @_;
+sub _exclude {
+    my ( $caller, $handle, @methods  ) = @_;
 
-    unless ( defined $handle->{'blacklist'} ) {
+    my $name='denylist';
```

Comment:
Why this variable for the attribute name?

Reply:
No reason other than to not hard code it. If the preference is to use a string I will - it would be more efficient.

Reply:
changed code based on @Leont 's recommendation